README.md

```diff
@@ -212,6 +234,36 @@ try {
 
 Replace `${ACCESS_KEY}` with yours obtained from Picovoice Console, `${MODEL_FILE}` with the default or custom trained model from [console](https://console.picovoice.ai/), and `${AUDIO_FILE_PATH}` with the path to the audio file.
 
+### React Native
+
+The Leopard React Native binding is available via [NPM](https://www.npmjs.com/package/@picovoice/leopard-react-native). Add it via the following command:
+
+```console
+yarn add @picovoice/leopard-react-native
+```
+
+Create an instance of the engine and transcribe an audio file:
+
+```typescript
+import {Leopard, LeopardErrors} from '@picovoice/leopard-react-native';
+
+const getAudioFrame = () => {
+  // get audio frames
+}
+
+try {
+  const leopard = await Leopard.create("${ACCESS_KEY}", "${MODEL_FILE}")
+  const transcript = await leopard.processFile("${AUDIO_FILE_NAME}")
+  console.log(transcript)
+} catch (err: any) {
+  if (err instanceof LeopardErrors) {
+    // handle error
+  }
+}
+```
+
+Replace `${ACCESS_KEY}` with yours obtained from Picovoice Console, `${MODEL_FILE}` with the default or custom trained model from [console](https://console.picovoice.ai/) and `${AUDIO_FILE_NAME}` with the name of the audio file. When done be sure to explicitly release the resources using `leopard.delete()`.
```

Comment:
`${AUDIO_FILE_NAME}` => `${AUDIO_FILE_PATH}` and mention that you need to absolute path